### PR TITLE
config: preprod to prod syncs are no longer automated

### PR DESCRIPTION
**Pull request by hectorj2f**
#### Summary

The `preprod-to-prod` syncs is no longer automated, we only support a manual trigger.

This issue is related to https://github.com/sigstore/root-signing/issues/893.

#### Release Note

preprod-to-prod syncs are no longer automated, only supporting a manual trigger.

### CODE REVIEW

**Review by lkatalin — APPROVED** [review]

**Review by Hayden-IO — CHANGES_REQUESTED** [review]
/hold

**Comment by hectorj2f** [comment]
@haydentherapper Let me know! When we can merge it.

File: .github/workflows/sync-preprod-to-prod.yml
Comment (Hayden-IO) [inline .github/workflows/sync-preprod-to-prod.yml:21]: Given this is manual now, we should remove the trigger_sync input too, and remove the "check sync conditions" step

**Comment by Hayden-IO** [comment]
Will merge after https://github.com/sigstore/root-signing/pull/916

**Review by Hayden-IO — APPROVED** [review]

**Comment by Hayden-IO** [comment]
Closing in favor of https://github.com/sigstore/root-signing/pull/928

**Pull request closed without merging**
